config.yml

```diff
@@ -118,6 +118,18 @@ java:
       VERSION: 11
       JAVA_VERSION: jdk-11.0.3_7-debian-slim
 
+kubectl:
+  "1.14":
```

Comment:
How about using 1.15, which has been released 6 days ago ?

Reply:
Upgraded to 1.15.1. I don't think it will break CI jobs.